### PR TITLE
Logging learning rate

### PR DESCRIPTION
**Pull request by leshanbog**
In this PR, learning rate is added to stats. Now it is easy to verify the LR scheduler work in wandb:

image

### CODE REVIEW

**Comment by LouisCastricato** [comment]
What does group number refer to?

File: trlx/trainer/accelerate_base_trainer.py [hunk 1]
```diff
@@ -281,6 +281,8 @@ def learn(self):
 
                     stats["time/forward"] = forward_time
                     stats["time/backward"] = backward_time
+                    for group_number, lr in enumerate(self.scheduler.get_last_lr()):
+                        stats[f"learning_rate/group_{group_number}"] = lr
```
Comment (jon-tow) [inline trlx/trainer/accelerate_base_trainer.py:285]: Currently, we're logging a ton of metrics: 
image
so to avoid adding extra clutter, I think it might be best to keep the LR dynamics tracked in `Charts` by using the following stats key instead: `stats[f"learning_rate_group_{group_number}"] = lr`

**Review by jon-tow — CHANGES_REQUESTED** [review]
Very cool!

File: trlx/trainer/accelerate_base_trainer.py (hunk 1 above)
Comment (leshanbog) [reply trlx/trainer/accelerate_base_trainer.py]: Agree. Moved it to charts

image

**Comment by leshanbog** [comment]
> What does group number refer to?

An index number of a particular [parameter group](https://pytorch.org/docs/stable/optim.html#per-parameter-options). In most cases there will be only one but we will also plot correctly in case of several groups

**Review by jon-tow — APPROVED** [review]
Thanks for the new feature! LGTM
